docs/chipflow-toml-guide.rst

```diff
@@ -0,0 +1,110 @@
+Intro to chipflow.toml
+======================
+
+The `chipflow.toml` file provides configuration for your design with the ChipFlow platform.
+
+Let's start with a typical example:
+
+.. literalinclude :: example-chipflow.toml
+   :language: toml
+
+.. testcode::
+   :hide:
+
+   # Assert that example-chipflow.toml matches the current config schema. If 
+   # this test fails, then its likely that the content in this file will need
+   # to be updated.
+   from chipflow_lib.cli import _parse_config_file
+   import os
+   root_dir = os.getcwd()
+   _parse_config_file(f"{root_dir}/docs/example-chipflow.toml")
+
+===========
```

Comment:
You don't need to have the top `=====`; it's present in all the examples but I feel like it adds too much visual noise, personally.

Reply:
I removed these